auto_nag/scripts/configs/tools.json

```diff
@@ -565,6 +565,23 @@
             "jdescottes@mozilla.com"
         ]
     },
+    "spambug":
+    {
+        "max_days_in_cache": 1,
+        "receivers":
+        [
+            "ehumphries@mozilla.com",
+            "mcastelluccio@mozilla.com"
```

Comment:
Actually, could you add sylvestre@mozilla.com back? Sylvestre was interested in getting the email too.